glslc/test/include.py

```diff
@@ -142,21 +151,23 @@ class VerifyNestedIncludeAmongSiblings(expect.StdoutMatch):
     """Tests #include inside #included sibling files."""
 
     environment = Directory('.', [
-        File('a.vert', '#include "b"\ncontent a\n'),
+        File('a.vert', '#version 140\n#include "b"\ncontent a\n'),
         File('b', 'content b\n#include "c"\n'),
         File('c', 'content c\n')])
 
     glslc_args = ['-E', 'a.vert']
 
     expected_stdout = \
-"""#extension GL_GOOGLE_include_directive : enable
+"""#version 140
+#extension GL_GOOGLE_include_directive : enable
 #line 0 "a.vert"
```

Comment:
I haven't modified any of the glslang #line processing or #include processing.
Are you objecting to the line number still being 0?  
I suspect that the #extension line is injected as a preamble that doesn't count toward line numbers in a.vert.  Also, #line number handling semantics changed in some version, I can't remember which.